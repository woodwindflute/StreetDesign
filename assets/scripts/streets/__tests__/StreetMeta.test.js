/* eslint-env jest */
import React from 'react'
import StreetMeta from '../StreetMeta'
import { render } from '../../../../test/helpers/render'

jest.mock('../../app/load_resources', () => {})
jest.mock('../../app/initialization', () => {})
jest.mock('../../preinit/app_settings', () => {})

describe('StreetMeta', () => {
  it('renders without crashing', () => {
    const { asFragment } = render(<StreetMeta />)
    expect(asFragment()).toMatchSnapshot()
  })
})
