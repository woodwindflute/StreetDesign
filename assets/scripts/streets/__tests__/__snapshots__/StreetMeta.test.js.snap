// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StreetMeta renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="street-metadata"
  >
    <span
      class="street-metadata-width"
    >
      <span
        class="street-width street-width-editable"
        title="Change width of the street"
      >
        0 m width
      </span>
    </span>
    <span
      class="street-metadata-map"
    >
      <a>
        Add location
      </a>
    </span>
  </div>
</DocumentFragment>
`;
